eng/service.proj

```diff
@@ -24,4 +24,14 @@
     <Error Condition="'$(Scope)' == '' and '@(ProjectReference)' == ''"
         Text="No Projects found with patttern [..\sdk\$(ServiceDirectory)\**\*.csproj], please make sure you have passed in the correct ServiceDirectory." />
   </Target>
+
+  <Target Name="GenerateAPIListing"
```

Comment:
In order to produce the api you are going to need to do a build so why introduce a new target? Instead I suggest just a property you set.

Reply:
Done